backend/src/main/resources/db/migration/V1_1__alter_Address_SubAddress.sql

```diff
@@ -0,0 +1,8 @@
+ALTER TABLE Address DROP CONSTRAINT address_subaddress_fkey;
```

Comment:
As your PR is not merged, you can make changes in your initial script. But you need to clean your local DB (your tables + flyway_schema_history - is storing migration data).